tweet-ui/src/main/java/com/twitter/sdk/android/tweetui/PlayerActivity.java

```diff
@@ -52,4 +58,28 @@ public void onDestroy() {
         playerController.cleanup();
         super.onDestroy();
     }
+
+    @Override
+    protected void onResume() {
+        super.onResume();
+
+        if(videoPaused) {
+            videoView.seekTo(videoPosition);
+            videoView.start();
+
+            // restart the VideoControlView
+            videoControlView.resume();
+        }
+    }
+
+    @Override
+    protected void onPause() {
+        super.onPause();
+
+        if(videoView.isPlaying()) {
+            videoPosition = videoView.getCurrentPosition();
```

Comment:
I think it's necessary to ensure the VideoView resumes playing at the position it was at when the activity was paused.